.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Terraform Cloud/Enterprise Troubleshooting and Feature Requests
+    url: mailto:tf-cloud@hashicorp.support
+    about: For issues and feature requests related to the Terraform Cloud/Enterprise platform, please contact tf-cloud@hashicorp.support
```

Comment:
I didn't realize this section existed! We could also add something here for the Terraform Registry. I can address that in a follow-up, but here's something basic:

```yaml
  - name: Terraform Registry Troubleshooting and Feature Requests
    url: mailto:terraform-registry@hashicorp.com
    about: For issues relating to the Terraform Registry and its website, please contact terraform-registry@hashicorp.com
```

Reply:
👍🏻 I'll let you handle as desired in all places (I think, just like this PR, we should have a common table like the GitHub New Issue UI in the templates themselves cause it's easy to miss!) separately! 💞